robot-server/robot_server/sessions/router/commands_router.py

```diff
@@ -0,0 +1,93 @@
+"""Router for /sessions endpoints."""
```

Comment:
Same as above. Maybe:
```suggestion
"""Router for /sessions commands endpoints."""
```